### PR TITLE
Final Submission

### PR DESCRIPTION
**Pull request by Mr-Darfler**

### CODE REVIEW

File: Class-Code/Elementary-CA/fcsmakers/sketch.js [hunk 1]
```diff
@@ -11,4 +11,6 @@ function setup() {
 
 function draw() {
     background(0);
+    fill(255)
```
Comment (Mr-Darfler) [inline Class-Code/Elementary-CA/fcsmakers/sketch.js]: Hey, wouldn't it be cool if the background were red and the ball was green

File: Class-Code/Elementary-CA/fcsmakers/sketch.js [hunk 2]
```diff
@@ -11,4 +11,6 @@ function setup() {
...
+    ellipse(frameCount,height/2,20,20)
```
Comment (Mr-Darfler) [inline Class-Code/Elementary-CA/fcsmakers/sketch.js]: This needs bigger. 20 px just doesn't show up well. however good use of the frameCount Variable

**Review by Mr-Darfler — COMMENTED** [review]
Great. Thanks for the changes. Adding @Jkohl1 for a second review

**Review by Jkohl1 — APPROVED** [review]
good stuff

**Review by gordoniii — APPROVED** [review]

**Comment by gordoniii** [comment]
g_that_boul

**Comment by Mr-Darfler** [comment]
Great Work @fcs-cs/cs2-2019

**Pull request merged**